src/lib/utils.ts

```diff
@@ -143,10 +143,13 @@ export function validateMetadata(object: Package, name: string): Package {
 export function combineBaseUrl(protocol: string, host: string | void, prefix?: string | void): string {
   let result = `${protocol}://${host}`;
 
-  if (prefix) {
-    prefix = prefix.replace(/\/$/, '');
+  if (prefix && prefix.length !== 0) {
```

Comment:
```suggestion
  if (prefix && prefix.length) {
```

Reply:
Already tried that:
```
Prefer using an optional chain expression instead, as it's more concise and easier to read.eslint(@typescript-eslint/prefer-optional-chain)
```

Reply:
What's the point of `prefix` property type `void`?

Because that's why I can't use the optional chain operator:
```ts
if (prefix?.length) {
```

Then it must be something like this:
```ts
if (typeof prefix === 'string' && prefix?.length) {
```

Reply:
Why not `_.isString()`? We have used `lodash` all over the place anyway, also @DanielRuf says is not a critical code, it is called once in a time.